src/Makefile.am

```diff
@@ -213,6 +213,7 @@ detect-http-uri.c detect-http-uri.h \
 detect-http2.c detect-http2.h \
 detect-icmp-id.c detect-icmp-id.h \
 detect-icmp-seq.c detect-icmp-seq.h \
+detect-icmp-hdr.c detect-icmp-hdr.h \
```

Comment:
can you rename to detect-icmpv4hdr.[ch] to match `icmpv6.hdr`? The other `detect-icmp-*.[ch]` apply to both ICMPv4 and ICMPv6.

Reply:
Will do.